scalafix-core/src/main/scala/scalafix/internal/config/ParserConfig.scala

```diff
@@ -13,16 +13,10 @@ case class ParserConfig(
     inlineKeyword: Boolean = false
 ) {
 
-  private val dialect = scala.meta.dialects.Scala213.copy(
```

Comment:
I don't see the point of writing our own definition of dialect. this one is not correct.
  implicit val Scala213 = Scala212
    .withAllowImplicitByNameParameters(true)
    .withAllowLiteralTypes(true)
    .withAllowNumericLiteralUnderscoreSeparators(true)
    .withAllowTryWithAnyExpr(true)

Reply:
I am not sure either - does it make sense to keep `ParserConfig` at all?

Reply:
IIRC, the motivation for the custom dialect was to allow as much as possible of Scala 3 syntax without introducing regressions for Scala 2 users. Some Scala 3 syntax can be enabled safely while other syntax like significant indent may change the parsed structure of Scala 2 programs.

Reply:
Ah, thanks! I understand better! Clever idea.

Reply:
Ah, thanks! I understand better! Clever idea.